src/enforcer.ts

```diff
@@ -357,6 +357,32 @@ export class Enforcer extends ManagementEnforcer {
 
     return res.filter((n) => !inherits.some((m) => n === m));
   }
+
+  /**
+   * getUsersForRoleInDomain gets the users from name and domain.
+   *
+   * @param name the user.
+   * @param domain the domain.
+   * @return the users that for role in domain.
+   */
+  public async getUsersForRoleInDomain(name: string, domain: string): Promise<string[]> {
+    const rm = this.rmMap.get('g');
+    if (rm) return rm.getUsers(name, domain);
+    throw new Error("RoleManager didn't exist.");
+  }
+
+  /**
+   * getRolesForUserInDomain gets the users from name and domain.
+   *
+   * @param name the user.
+   * @param domain the domain.
+   * @return the roles that for user in domain.
+   */
+  public async getRolesForUserInDomain(name: string, domain: string): Promise<string[]> {
+    const rm = this.rmMap.get('g');
```

Comment:
You should call the https://github.com/casbin/casbin.js/pull/211/files#diff-0a73a3b16c456ebf5c6ac8b0a826dc308426af2f0cb5cdb9bad897ef1ee3f9d4R97.